### PR TITLE
https://bugs.swift.org/browse/SR-6812

### PR DESCRIPTION
**Pull request by chnmrc**
Resolve a bug that affects ARMv7 system
https://bugs.swift.org/browse/SR-6812

### CODE REVIEW

**Comment by hpux735** [comment]
Thanks @chnmrc !  This seems pretty straight-forward to me.

**Comment by millenomi** [comment]
I assume this just moves the CF types to the memory layout Swift objects have on 32 bit platforms? Is it the same on other 32 bit platforms or does it need a specific armv7 guard?

**Comment by hpux735** [comment]
That's a good question.  We don't have testing on other non-supported 32-bit platforms.  As there aren't any other 32-bit platforms in community CI, I can't be sure who represents those platforms, if they're being used at all.

File: CoreFoundation/Base.subproj/CFRuntime.h [hunk 1]
```diff
@@ -195,10 +195,18 @@ typedef struct __CFRuntimeBase {
     uintptr_t _cfisa;
     uintptr_t _swift_rc;
     // This is for CF's use, and must match _NSCFType layout
+#if __LP64__
```
Comment (parkera) [inline CoreFoundation/Base.subproj/CFRuntime.h]: Presumably `LLP64`, too.

Comment (chnmrc) [reply CoreFoundation/Base.subproj/CFRuntime.h]: Yes, for Windows also, maybe we must add an || condition.

Comment (chnmrc) [reply CoreFoundation/Base.subproj/CFRuntime.h]: I agree

Comment (parkera) [reply CoreFoundation/Base.subproj/CFRuntime.h]: Looks like the .c file was updated but not the .h file?

Comment (chnmrc) [reply CoreFoundation/Base.subproj/CFRuntime.h]: Is that a question to me?

**Comment by parkera** [comment]
@swift-ci please test

**Comment by millenomi** [comment]
@swift-ci please test and merge

**Comment by spevans** [comment]
@swift-ci please test and merge

**Pull request merged**